apps/bfd-pipeline/bfd-pipeline-rda-grpc-apps/pom.xml

```diff
@@ -16,8 +16,8 @@
     </description>
 
     <properties>
+        <checkstyle.skip>false</checkstyle.skip>
```

Comment:
Seems like this still is here; please remove this for the PR.

Reply:
removed